lib/ZwaveClient.ts

```diff
@@ -3372,16 +3372,16 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 			valueId.propertyKey = args.eventLabel as string
 
 			data = this._parseNotification(args.parameters)
-		} else if (
-			ccId === CommandClasses['Entry Control'] ||
-			ccId === CommandClasses['Multilevel Switch']
-		) {
+		} else if (ccId === CommandClasses['Entry Control']) {
 			valueId.property = args.eventType as string
 			valueId.propertyKey = args.dataType as string
 			data =
 				args.eventData instanceof Buffer
 					? utils.buffer2hex(args.eventData)
 					: args.eventData
+		} else if (ccId === CommandClasses['Multilevel Switch']) {
+			valueId.property = args.eventType as string
```

Comment:
`args.eventType`  is not a string, writing `as string` behind that doesn't change it.

It is one of these two enum values:
https://github.com/zwave-js/node-zwave-js/blob/3ef1368bd1f2fd1e41ceac43bc9f89e0d1ffe550/packages/zwave-js/src/lib/commandclass/MultilevelSwitchCC.ts#L50-L51

This way you'll end up with a property of `4`  or `5`  (number, not string!)

Reply:
You might want to use `getEnumMemberName` here

Reply:
done